encrypted_merlin.md

```diff
@@ -0,0 +1,91 @@
+# Encrypted Merlin
+
+It is possible to configure merlin to encrypt the packets sent over the merlin port (default 15551).
+This can be useful if different OP5 Monitor nodes need to communicate over the internet, for example if you have a remote poller.
+For nodes which are placed within the same secure local network, encryption is often unnecessary, and introduced an unnecessary performance penalty.
+
+Merlin uses public key encryption, which requires setup of both a private and public key for each node.
+
+The encryption used is:
+
+Key agreement protocol: ECDH
+Encryption: XSalsa20
+Authentication: Poly1305 MAC
+
+# Setup
+
+## Generate and copy keys
+To setup encryption, you must first generate a key pair on every node involved in the encrypted communication.
+
+This is done using the command `mon merlinkey generate`. This will generate two files:
+key.priv and key.pub at /opt/monitor/op5/merlin by default, or you can use --path=/preferred/path/here
+
+When the keys have been created, the public key only (key.pub) needs to be copied to the relevant servers.
+The private key (key.priv) should never be shared.
```

Comment:
I think that this line should go to the end of the paragraph and be prefixed with "Important:" in bold. I think it is easier to read the text when we talk about copying and then in the next section we give an example of how it can be copied and not have the part of private key in between.

Reply:
You mean you want it after the `scp` commands?